ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala

```diff
@@ -80,8 +80,10 @@ class CommandTrackerFlowTest
       Some(
         Commands(
           commandId = commandId,
-          deduplicationTime =
-            dedupTime.map(t => com.google.protobuf.duration.Duration(t.getSeconds)),
+          deduplication = dedupTime
+            .map(t => com.google.protobuf.duration.Duration(t.getSeconds))
+            .map(Commands.Deduplication.DeduplicationTime)
```

Comment:
Please fuse these maps.

I think it's reasonable to alias `com.google.protobuf.duration.Duration` as `DurationProto` in the imports for terseness.